### PR TITLE
Azure - document disaster recovery

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Change Azure storage shape when disaster recovery is enabled

### CODE REVIEW

File: stories/topics/con-azure-disaster-recovery.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+= Disaster Recovery
+
+When you deploy {AAPonAzureNameShort}, you must enable or disable Disaster Recovery in the `Business Continuity` tab of the form.
+There is no default setting for Disaster Recovery.
+
+The Disaster Recovery feature incurs additional Azure infrastructure costs.
+See xref:con-azure-infrastructure-usage_azure-intro[Azure infrastructure usage] for details of the Service Shape of the Storage account.
+
+
```
Comment (aldato) [inline stories/topics/con-azure-disaster-recovery.adoc:11]: We should add that once the AAP is deployed, customers who'd like to enable DI should contact customer support in order to do so.

Comment (aldato) [reply stories/topics/con-azure-disaster-recovery.adoc:11]: Shall we add here a link to the KCS which explains Disaster Recovery in detail? 

https://access.redhat.com/articles/7010302

File: stories/topics/con-azure-infrastructure-usage.adoc [hunk 2]
```diff
@@ -20,7 +20,10 @@ Azure Database for PostgreSQL:: An Azure database service used for {PlatformName
 **Service Shape:** 1 TB
 Storage account:: The Azure service is used for file and block storage for local storage of projects, containers, etc.
```
Comment (EMcWhinn) [inline stories/topics/con-azure-infrastructure-usage.adoc:21]: As per SG, I would avoid using "etc.", instead:

"The Azure service is used for file and block storage such as local storage of projects and containers."

File: stories/topics/con-azure-disaster-recovery.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[id="con-azure-disaster-recovery_{context}"]
+
+= Disaster Recovery
+
+When you deploy {AAPonAzureNameShort}, you must enable or disable Disaster Recovery in the `Business Continuity` tab of the form.
+There is no default setting for Disaster Recovery.
+
+If you want to enable Disaster Recovery on an existing instance of {AAPonAzureNameShort}, contact Red Hat customer support.
```
Comment (EMcWhinn) [inline stories/topics/con-azure-disaster-recovery.adoc:8]: Just wondering here if we should hyperlink "Red Hat customer support" directly to support, or is this fairly common knowledge?

**Comment by EMcWhinn** [comment]
Just added 2 very small comments, otherwise LGTM.

File: stories/topics/con-azure-disaster-recovery.adoc (hunk 3 above)
Comment (ariordan-redhat) [reply stories/topics/con-azure-disaster-recovery.adoc:8]: I thought it better not to link, as different customers might have different contact points with Red Hat.

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**